R/get_test_data.R

```diff
@@ -28,32 +28,25 @@ get_test_data <- function(recipe, x){
   }
   ## CHECK if it is epi_df?
 
-
-  # initialize vector to hold max lags for each variable
-  max_lags <- c()
-  for (i in seq_along(recipe$steps)) {
-    if ("lag" %in% names(recipe$steps[[i]])) {
-      max_lags <- append(max_lags, max(recipe$steps[[i]]$lag))
-    }
-  }
+  max_lags <- max(map_dbl(recipe$steps, ~ max(.x$lag %||% 0)))
```

Comment:
definitely a good idea to change this. I'll avoid evergrowing vectors in the future too, thank you!